README.md

```diff
@@ -87,12 +87,53 @@ jsep.removeIdentifierChar('@');
 
 ### Plugins
 JSEP supports defining custom hooks for extending or modifying the expression parsing.
-All hooks are bound to the jsep instance and called with a single argument and return void.
-The 'this' context provide access to the internal parsing methods of jsep
+Plugins are registered by calling `jsep.plugins.register(the_plugin)`.
+
+#### JSEP-provided plugins:
+* `ternary`: Built-in by default, adds support for ternary `a ? b : c` expressions
```

Comment:
Should we add some wording to indicate that we plan to eventually remove this from core jsep and make it an optional plugin?

Reply:
Yeah probably worth mentioning ... depends on when/if we want a breaking change release?